parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -844,6 +844,22 @@ ensureGPUDrivers() {
     fi
 }
 
+# TODO: this is a temporary ubuntu-only HACK until we get a driver
+ensureAMDGPUDrivers() {
+    echo "Installing AMD GPU drivers"
+
+     # delete amdgpu module from blacklist
+    sudo sed -i '/blacklist amdgpu/d' /etc/modprobe.d/blacklist-radeon-instinct.conf
```

Comment:
can we get a better explanation why we have that ? why understanding was this was required on a VM to enable the access of the physical hardware on the bareVM

Reply:
I'd like to know as well why we need it. It took me a lot of time to figure out why drivers didn't work.

I think in the image provided by Canonical it's blacklisted, but nothing explains why it's there.